src/main/java/uk/gov/hmcts/dts/fact/controllers/admin/AdminCourtAreasOfLawController.java

```diff
@@ -51,6 +64,12 @@ public ResponseEntity<List<AreaOfLaw>> getCourtAreasOfLaw(@PathVariable String s
         return ok(adminCourtAreasOfLawService.getCourtAreasOfLawBySlug(slug));
     }
 
+    /**
+     * Update the areas of law for a court.
+     * @param slug Court slug
+     * @param areasOfLaw A list of areas of law
+     * @return A list of court areas of law
+     */
```

Comment:
I think the Authentication object may have been missed from the params

Reply:
added missing bit